tests/test_annotation.yml

```diff
@@ -24,8 +26,9 @@
     - snpeff
     - vep
   files:
+    #TODO @Maxime: should snpeff alone here be published I find it a bit cofusing to have snpeff + merged but not VEP alone
```

Comment:
Good point.
I guess if we specify `merge`, we should only have `snpEff_VEP`.
Only `snpeff,merge` should give out that output.

Reply:
But can be fixed in a separate PR